File: app/src/main/java/ua/fvadevand/reminderstatusbar/FakeDataUtils.java
package ua.fvadevand.reminderstatusbar;

import java.util.ArrayList;
import java.util.List;

import ua.fvadevand.reminderstatusbar.data.models.Reminder;

public class FakeDataUtils {
    private FakeDataUtils() {
    }

    public static List<Reminder> getReminderList() {
        List<Reminder> reminderList = new ArrayList<>();
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));
//        reminderList.add(new Reminder(R.drawable.ic_add_24dp, "Hello", "How Are You", 54654854));

        return reminderList;
    }

    public static Reminder getReminder() {
        return new Reminder();
    }
}

File: app/src/main/java/ua/fvadevand/reminderstatusbar/FakeDataUtils.kt
package ua.fvadevand.reminderstatusbar

import ua.fvadevand.reminderstatusbar.data.models.Reminder
import java.util.ArrayList

object FakeDataUtils {

    val reminderList: List<Reminder>
        get() = ArrayList<Reminder>().apply {
            add(Reminder(
                    "Reminder",
                    null,
                    R.drawable.ic_notif_id_card,
                    System.currentTimeMillis(),
                    false))
            add(Reminder(
                    "Reminder",
                    "Subscription",
                    R.drawable.ic_notif_file,
                    System.currentTimeMillis(),
                    false))
            add(Reminder(
                    "Reminder",
                    null,
                    R.drawable.ic_notif_like,
                    System.currentTimeMillis(),
                    false))
        }

    val reminder: Reminder
        get() = Reminder(
                "Reminder",
                null,
                R.drawable.ic_notif_briefcase,
                System.currentTimeMillis(),
                false
        )
}
